app/[...slug]/page.tsx

```diff
@@ -26,141 +19,11 @@ const NodePage = async (props: PageProps & {previewMode?: true}) => {
   if (routeRedirect) redirect(routeRedirect)
   if (!entity) notFound()
 
-  const menuItems = await getMenu()
-
-  const fullWidth =
-    (entity.__typename === "NodeStanfordPage" && entity.layoutSelection?.id === "stanford_basic_page_full") ||
-    (entity.__typename === "NodeSulLibrary" && entity.layoutSelection?.id === "sul_library_full_width")
-
-  const sulSidebar =
-    (entity.__typename === "NodeStanfordPage" && entity.layoutSelection?.id === "sul_side_nav") ||
-    (entity.__typename === "NodeStanfordNews" && entity.layoutSelection?.id === "sul_news_side_nav") ||
-    (entity.__typename === "NodeSulLibrary" && entity.layoutSelection?.id === "sul_branch_side_nav")
-
   return (
     <main id="main-content" className="mb-50">
-      <article aria-labelledby={entity.id}>
```

Comment:
You should add the `<article>` tag back to the page displays now. At least the basic page and person pages need it.